api/src/repositories/survey-repository.ts

```diff
@@ -1100,4 +1101,188 @@ export class SurveyRepository extends BaseRepository {
 
     return response.rowCount;
   }
+
+  /**
+   * @TODO jdsdoc
+   *
+   * @TODO should probably simplify this method to use knex? And accept [] as a valid response? ...
+   *
+   * @param {number} surveyId
+   * @return {*}  {Promise<GetPartnershipsData['indigenous_partnerships'][]>}
+   * @memberof SurveyRepository
+   */
+  async getIndigenousPartnershipsBySurveyId(
+    surveyId: number
+  ): Promise<GetPartnershipsData['indigenous_partnerships'][]> {
+    defaultLog.debug({ label: 'getIndigenousPartnershipsBySurveyId', surveyId });
+
+    const sqlStatement = SQL`
+      SELECT
+        fn.first_nations_id as id,
+        fn.name as first_nations_name
+      FROM
+        survey_first_nation_partnership sfnp
+      LEFT OUTER JOIN
+        first_nations fn
+      ON
+      sfnp.first_nations_id = fn.first_nations_id
+      WHERE
+      sfnp.survey_id = ${surveyId}
+      GROUP BY
+        fn.first_nations_id,
+        fn.name;
+    `;
+
+    const response = await this.connection.query(sqlStatement.text, sqlStatement.values);
+
+    const result = response?.rows;
```

Comment:
You should be able to drop the question mark as rows will always be defined and an array.

For the if statement below, rows will always be defined, so we should either remove that check, or make it a `response.rowCount` check instead.

Reply:
Same for the other new functions below*

Reply:
Will do.